src/plugins/data/public/ui/filter_bar/filter_bar.tsx

```diff
@@ -165,11 +175,17 @@ function FilterBarUI(props: Props) {
   }
 
   function onToggleAllNegated() {
+    if (props.trackUiMetric) {
+      props.trackUiMetric(METRIC_TYPE.CLICK, `${props.appName}:filter_invertInclusion`);
+    }
```

Comment:
@lizozom there are more functions here like `onDisableAll`, `onPinAll`, since this PR started to add metric to some of them this functions, I wonder if it makes sense to add metric to all of them?